xml/hwbp_registers.xml

```diff
@@ -220,6 +220,18 @@
         hold all valid virtual addresses but it need not be capable of holding
         other values.
 
+        If the A extension is supported, then trigger behavior is as follows
+        for the load and store bits:
+        \begin{tightlist}
+        \item LR instructions are loads
+        \item successful SC instructions are stores
+        \item it is \unspecified\ whether failing SC instructions are stores or not
+        \item Each AMO instruction is both a load and a store
+        \end{tightlist}
+
+        If the destination register of any load or AMO is x0 then it is
+        \unspecified\ whether a load data trigger will match.  Whether store
+        data triggers match on AMOs is \unspecified.
```

Comment:
Is it clear enough what is meant by "load data trigger" and "store data trigger"?  It is to me but it might be more precise to say "... a trigger with \FcsrMcontrolSelect = 1".

Reply:
I think it would be nice to have a glossary with terms like this.  Every time I read a passage like that I have to go look up what select=1 or timing=0 or action=1 means.  The descriptive terms would make it much easier to read.

That being said, it's a non-trivial effort which is beyond the scope of this particular PR so I'll take your advice.